src/WInterop.Desktop/Theming/Themes.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Jeremy W. Kuhne. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using WInterop.Errors;
+using WInterop.Support;
+
+namespace WInterop.Windows
+{
+    public static partial class Themes
+    {
+        public static unsafe string GetCurrentThemeName()
+        {
+            HResult result;
+            Span<char> themeName = stackalloc char[Paths.MaxPath];
+            fixed (char* theme = themeName)
+            {
+                result = Native.Imports.GetCurrentThemeName(theme);
```

Comment:
`Native.Imports.GetCurrentThemeName(theme).ThrowIfFailed();` is what I'm typically doing.